### ISSUE TITLE
Installing on CentOS 7 and SELinux disabled

### ISSUE THREAD

**Issue by sbelyanin** [brianshumate/ansible-nomad#58]
Hello.

When i starting playbook on host whith centos 7 and already disabled SELinux:

```bash
cat /etc/selinux/config 

# This file controls the state of SELinux on the system.
# SELINUX= can take one of these three values:
#     enforcing - SELinux security policy is enforced.
#     permissive - SELinux prints warnings instead of enforcing.
#     disabled - No SELinux policy is loaded.
SELINUX=disabled
```
and:

```bash
setenforce 0 || echo $? 
setenforce: SELinux is disabled
1
```
i get the following:

```bash
TASK [brianshumate.nomad : Stop SELinux enforcing] ******************************************************************
fatal: [myhost]: FAILED! => {"changed": true, "cmd": "setenforce 0", "delta": "0:00:00.005533", "end": "2019-05-22 17:02:33.749074", "msg": "non-zero return code", "rc": 1, "start": "2019-05-22 17:02:33.743541", "stderr": "setenforce: SELinux is disabled", "stderr_lines": ["setenforce: SELinux is disabled"], "stdout": "", "stdout_lines": []}

PLAY RECAP **********************************************************************************************************

**Comment by 5-sigma** [comment]
This issue could be resolved by switching tasks/selinux.yml to use the [ansible selinux module](https://docs.ansible.com/ansible/latest/modules/selinux_module.html).


Subsequently I would like to sugest to reboot hosts before proceeding after SELinux was disabled.
Creating volumes in docker with SELinux running in permissive can cause incompatibility when starting containers once the host has rebooted, because of remaining SELinux options in mounts.

I can create a pull request for this if you want.

**Comment by brianshumate** [comment]
Thanks for the heads up; this was resolved in v1.8.7.
